src/Combobox.jsx

```diff
@@ -148,7 +155,7 @@ var ComboBox = React.createClass({
       , afterListComponent, searchTerm, onChange
       , listComponent: List } = this.props;
 
-    List = List || (groupBy && GroupableList) || PlainList
+    List = List || _resolveListGroupable(groupBy) || PlainList
```

Comment:
`DropdownList` and `Combobox` take a `listComponent` prop. What do you think of exposing `MultiGroupList` from the package then passing `MultiGroupList` to `listCompoent` when you instaniate a new `Combobox` like so

```
import { MultiGroupList, Combobox } from 'react-widgets';

render() {
  <Combobox listComponent={MultGroupList} />
}
```

Naturally extending the functionality using the given api instead of adding checks for our special case.